db/models.py

```diff
@@ -50,3 +60,67 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField()
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE,
+        related_name="orders"
+    )
+
+    class Meta:
+        ordering = ("-created_at",)
+
+    def save(self, **kwargs) -> None:
+        if not self.created_at:
+            self.created_at = datetime.datetime.now()
+
+        super(Order, self).save(**kwargs)
+
+    def __str__(self) -> str:
+        return self.created_at.strftime("%Y-%m-%d %H:%M:%S")
+
+
+class User(AbstractUser):
+    pass
+
+
+class Ticket(models.Model):
+    movie_session = models.ForeignKey(
+        to=MovieSession, on_delete=models.CASCADE, related_name="tickets")
+    order = models.ForeignKey(
+        to=Order, on_delete=models.CASCADE, related_name="tickets")
```

Comment:
we don't add multiple parameters on the same line.
Either they are on the same lvl that ( bracket or 1 parameter - 1 line